tests/textbook_test.py

```diff
@@ -269,9 +220,4 @@ def test_invalid_department_code(self):
         self.assertRaises(
             ValueError, textbook.get_textbook, TERM, "TEST", "000", "EXIST", None
         )
-=======
         self.assertEqual(textbook._filter_dictionary(test_dict, test_key), {"a": 1, "c": 3})
```

Comment:
This line should've been deleted when you cleaned up this merge conflict, since it's part of the version that you deleted

Reply:
I deleted that last line  "self.assertEqual(textbook._filter_dictionary(test_dict, test_key), {"a": 1, "c": 3})" if that is what you were referring to.